cypress/support/ShowPage.js

```diff
@@ -1,7 +1,7 @@
 export default (url, initialField = 'title') => ({
     elements: {
         body: 'body',
-        field: name => `.ra-field-${name} > div > div > span`,
+        field: name => `.ra-field-${name} > p > span`,
```

Comment:
We really should stop this and use `@testing-library/cypress`. Not in this PR though